src/main/java/ua/lviv/iot/lab4/expression/Regular.java

```diff
@@ -0,0 +1,46 @@
+package ua.lviv.iot.lab4.expression;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Regular {
+
+    public String switchWords() {
```

Comment:
Please provide a better method name. How exactly are the words switched? This is important because of the same reason I commented on the class name previously.

Reply:
Thank you for your review. Naming is fixed.